Tasks/ANT/ant.ps1

```diff
@@ -108,9 +108,15 @@ if ($jdkPath)
 }
 
 $buildRootPath = Split-Path $antBuildFile -Parent
-$reportDirectoryName = [guid]::NewGuid()
+$reportDirectoryName = "ReportDirectory"
 $reportDirectory = Join-Path $buildRootPath $reportDirectoryName
 
+if(Test-Path $reportDirectory)
```

Comment:
as discussed this is not the full fix. if there r multiple maven tasks if will still be a problem. or if build completes next build will come delete these files. shuldt we wait for artifact upload to be completed. what happens to publish artifacttask?